recipe/meta.yaml

```diff
@@ -26,7 +27,6 @@ requirements:
   run:
     - python
     - libmo_unpack
-    - {{ pin_compatible('numpy') }}
```

Comment:
Not sure I've fully understood this yet.

https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#pinning-at-the-recipe-level

Reply:
Is this version ready for numpy 2.0? If not, it is better to re-try #35 first and keep this line. I merge the bot PR b/c it was correct as far as I can tell and I missed this one in draft. I'll restart both the numpy 2.0 and Python 3.13 migrations to see if they pass with the new version.

Reply:
This version is ready for NumPy v2 (that's the only change actually).

Does `{{ pin_compatible('numpy') }}` mean that previously released versions will stick with NumPy v1? If so that would be great!

Reply:
It will stick with the compatible numpy that it was built. However, with np 2.0, we no longer need that b/c the ABI is more consistent, so a simple `numpy >=2` on build and `numpy` on run would do. Let's wait for #35 to be re-issued for a proper migration. If that doesn't happen, we can do it manually.